src/POM/HomePage.js

```diff
@@ -0,0 +1,27 @@
+const {By} = require('selenium-webdriver');
```

Comment:
This file should be in a test folder - for example test/ui/pom/HomePage.js

Reply:
done